### ISSUE TITLE
Add NonbondedForce.{set|get}IncludeReciprocalSpace

### ISSUE THREAD

**Issue by jchodera** [openmm/openmm#4326]
`NonbondedForce` has a `{set|get}IncludeDirectSpace()` method that allows us to turn on/off direct-space interactions, but it would also be useful to have a `{set|get}IncludeReciprocalSpace()` as well for cases when one needs to easily assess the impact of omitting reciprocal space contributions to the energy without having to construct elaborate Custom forces.

**Comment by peastman** [comment]
Can you elaborate on the use cases?  Remember that reciprocal space contributes even to very short range interactions.  There's pretty much no situation when you would get accurate results by telling it to use PME but omit the reciprocal space part.

**Comment by jchodera** [comment]
I think you and I are both believers in the benefits of PME, but that does not dismiss the fact that it is one of the previously proposed short-ranged electrostatic schemes proposed as an alternative to periodic electrostatics. See, for example, [Stenqvist and Lund 2019](http://doi.org/10.1039/c9cp03875b), Table 1:
![image](https://github.com/openmm/openmm/assets/3656088/694618a5-7122-4b53-9390-6f7afe959dc3)
Or [timemachine](https://github.com/proteneer/timemachine), which uses this electrostatics model.

The most direct use case for us right now us that it would make it simple to omit the reciprocal-space contribution to electrostatics (without a ton of code rewriting) to perform quantitative experiments on the impact of omitting the reciprocal-space terms on free energy calculations. A second use case for us is enabling us to add back the reciprocal space contribution in reweighting schemes. A third use case would be the simplification of some multiple timestep methods that would otherwise require implementation of Custom forces to handle things like solute-solvent splitting.

**Comment by peastman** [comment]
Ok, sounds reasonable.

**Comment by peastman** [comment]
Note that you can do something very similar by just putting the reciprocal space part into a different force group.

**Comment by jchodera** [comment]
True, but this would also require some extensive changes to all the integrator handling in `openmmtools`, rather than a one-line change that would be easy if we supported the symmetry of `NonbondedForce.{set|get}Include{Direct|Reciprocal}Space()`
